test/membrane_rtc_engine/hls_endpoint_test.exs

```diff
@@ -166,15 +167,70 @@ defmodule Membrane.RTC.HLSEndpointTest do
       assert_receive({:cleanup, _cleanup_function, ^stream_id})
       refute_received({:cleanup, _cleanup_function, ^stream_id})
     end
+
+    test "check if number of headers is reduced to 1 when resolution is not stable", %{
+      rtc_engine: rtc_engine
+    } do
+      file_endpoint_id = "transcoding-file-endpoint-id"
+
+      file_name = "sample.h264"
```

Comment:
Let's call this file in a more meaningful way e.g. `variable_framerate.h264`.

Reply:
Fixed.